worldmap/worldmap.js

```diff
@@ -662,7 +662,10 @@ map.on('moveend', function() {
 var addmenu = "<b>Add marker</b><br><input type='text' id='rinput' autofocus onkeydown='if (event.keyCode == 13) addThing();' placeholder='name (,icon, layer, colour, heading)'/>";
 var rightmenuMap = L.popup({keepInView:true, minWidth:250}).setContent(addmenu);
 
-var rclk;
+var rclk = {
```

Comment:
I think maybe just initialising it to `{}` should be enough - as that way rclk.lat and .lon will be undefined (but at least undefined properties of an object) - so then when they are assigne later on as ...`lat:rclk.lat, lon:rclk.lng`... then lat and lon will be null and won't exist in the send - rather than sending 0,0 which is not a valid thing to send.

Reply:
True. will push a new PR